container/container.go

```diff
@@ -248,6 +248,10 @@ func (w *ContainerWorker) finalizeDeployment(agreementId string, deployment *con
 		vols := make(map[string]struct{})
 		for _, bind := range service.Binds {
 			containerVol := strings.Split(bind, ":")
+			// Create all host-paths before docker creates them with root user so we'll be able to clear them without root rights
+			if err := os.MkdirAll(containerVol[0], 0700); err != nil {
```

Comment:
There are several issues with this part of the change:
1) See https://github.com/open-horizon/anax/blob/master/doc/deployment_string.md. Binds can be files/directories or docker volumes, so you have to account for that here and skip the volume binds.
2) I am concerned that this fix (the 0700 aspect) will break the mac use case. I dont have a mac so I cant test it. The key to testing the mac scenario is getting this script: https://github.com/open-horizon/anax/blob/master/anax-in-container/horizon-container to run the anax-in-container (which you will have to build with your fix in it). Can you test this? If you have trouble, @linggao can help you.
3) I did test this fix on linux, and noticed that hzn dev service stop did NOT clean up the host side files if the service had a file binding in the deployment. I can show you how to set that up if you need help.
4) File bindings can be read-only (ro) and so the file mask on the mkdir needs to reflect that.

Given all the above concerns, this might be enough to consider breaking this fix out into it's own issue so we fix the reported now. Let me know what you think.

Reply:
Updated the PR